src/lib/components/Toast.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition';
+	import CloseToastIcon from './icons/CloseToastIcon.svelte';
+	import WarningIcon from './icons/WarningIcon.svelte';
+  import { createEventDispatcher } from 'svelte';
+
+  const dispatch = createEventDispatcher();
+
+</script>
+
+<div class="toast-dialog" role="alert" transition:fade>
+	<WarningIcon />
```

Comment:
Optional: To allow for different icons we can extract this into a prop as well
```typescript
export let icon = WarningIcon
```
and then here we can consume whatever icon gets passed in
```svelte
<svelte:component this={WarningIcon} />
```